### DISCUSSION TITLE
Can you share your cursor?

### DISCUSSION THREAD

**Discussion by Dhyfer1** [uxbox/uxbox#584] [General]
I was just looking at Penpot's home page and I saw the cursor you are showing at in the following picture

![01-design-feature](https://user-images.githubusercontent.com/50301201/107085837-0aaa0700-67c7-11eb-8a5b-d6207f6306bf.gif)

I like that cursor and I want to know if you can share it? or where I can download it?
That's all, I look forward to your reply. 🙂

**Comment by elhombretecla** [comment]
Hi!!! Of course, they're all available at our design system in Penpot.

Once you create a new user you will get 2 new files. At "Penpot- Design System" you will find the icon set we are currently using in Penpot. Including that cursor (page "Assets", icon "Pointer-1"). You can export it from there to SVG, PNG o JPG... 

Please tell us if you have any problem to find it :)

**Reply by Dhyfer1** [reply]
Thanks, but I found the cursor in the penpot code and it is now my favorite cursor. Now I am using it. 😀

**Comment by Alotor** [comment]
You can check all the cursors in this folder: https://github.com/penpot/penpot/tree/develop/frontend/resources/images/cursors

The one you're refering to is "pointer-inner.svg"

**Reply by Dhyfer1** [reply]
Thank you very much, I found the file and now I am using the cursor, thank you. :)
